offchain/advance-runner/Cargo.toml

```diff
@@ -3,12 +3,14 @@ name = "advance-runner"
 edition.workspace = true
 license.workspace = true
 version.workspace = true
+build = "../build-info/src/build.rs"
```

Comment:
Can we move this logic to the log crate and all other modules import it by default.

Reply:
We can move it there. Will test if `build = "../build-info/src/build.rs"` can be a "workspace" config, or would be better for all services have the same solutions as `inspect-server` that has a custom `build.rs` script that I modified to add the `built` procedure

Reply:
Just to confirm, @gligneul, do you meaning the whole logic of building `built.rs`, and printing the build info a single place and somehow reusing it in every module to avoid the repeated code?